config/configuration.json

```diff
@@ -444,6 +444,23 @@
       "difficulty": "HARD"
     }
   },
+  "RoadNameGapCheck": {
+    "tag.filters":[
```

Comment:
Are these tag filters being used?

Reply:
We are not using it. I thought its necessary to add this filters in json file. But, will take it out.